### DISCUSSION TITLE
Color correction function

### DISCUSSION THREAD

**Discussion by danielleunglol** [light-and-ray/sd-webui-replacer#87] [Ideas]
![image](https://github.com/light-and-ray/sd-webui-replacer/assets/70418507/b2b9a000-cc08-4a90-846b-26a5569118bf)

There is a color correction function for the default Stable Diffusion img2img. I wonder if adding this function to the replacer is possible. I think it would be a great addition to the extension since it will give a more realistic result.

**Comment by light-and-ray** [comment]
I'm not sure you understand what this option do. If I add, the objects under mask will affect whole result image, not even only masked part. This option is good for img2img with no inpainting

You can add this feature in extras tab https://github.com/light-and-ray/sd-webui-color-correction-extras (my other extension) and use it there. I use it in this way more often then in img2img tab
